R/config.R

```diff
@@ -427,6 +427,7 @@ parse_env_vars <- function() {
     "SNOWFLAKE_WAREHOUSE",
     "SNOWFLAKE_AUTHENTICATOR",
     "SNOWFLAKE_PRIVATE_KEY_PATH",
+    "SNOWFLAKE_PRIVATE_KEY_FILE",
```

Comment:
Is there test coverage for the environment variables (both against default and against other named connections)?